### DISCUSSION TITLE
Is it possible to code sign the package created using customtkinter with Auto-py-to-exe?

### DISCUSSION THREAD

**Discussion by DyNgPinasOfficial** [TomSchimansky/CustomTkinter#2326] [Q&A]
I had created a desktop app with the UI is based on customtkinter and I want to do a code signing in order to indicate that the app was created by me.

**Comment by Akascape** [comment]
@DyNgPinasOfficial First you need to convert the app to exe, check this thread if you need help: #939
Auto-py-to-exe or pyinstaller is just a tool to convert py to exe. Signing the code/exe is a different topic.

**To sign an exe file you need an official code signing certificate.**

There are tool like microsoft signtool which can help you sign the executable with the certificate.

I guess here you can find more information about this: https://stackoverflow.com/questions/252226/signing-a-windows-exe-file
